flock_agent/daemon/daemon.py

```diff
@@ -467,3 +474,8 @@ def log(self, request, response, time):
                     "autoupdate_loop",
                     f"Exception while checking for updates: {e}",
                 )
+                await asyncio.sleep(30)
+
+    def cleanup(self, sig=None, frame=None):
```

Comment:
What are sig and frame?

Reply:
Oh, those aren't needed anymore. Initially I was trying to force this method to run when the daemon was asked to quit by the OS (like, if the computer is rebooting) using basically this: https://stackoverflow.com/a/31464349

But then after rebooting and seeing it didn't get called, I decided to remove the `signal` code, but I left in those args.